cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,29 +1,34 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get(
-        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
-    ).click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
+import menuContentPage from "../page/menu-content.page";
+import productListPage from "../page/products-list.page";
+import shoppingCartPage from "../page/shoping-cart.page";
+import loginPage from "../page/login.page";
+import addressStepPage from "../page/address-step.page";
+import shippingStepPage from "../page/shipping-step.page";
+import paymentStepPage from "../page/payment-step.page";
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
 
-    cy.get("[name='SubmitLogin']").click();
+    productListPage.clickAddToCartBtn();
+    productListPage.clickProceedToCheckout();
 
-    cy.get(".cart_navigation button").click();
+    shoppingCartPage.clickProceedToCheckout();
 
-    cy.get("[type='checkbox']").click();
+    loginPage.typeEmailInput();
+    loginPage.typePasswordInput();
+    loginPage.clickSignInBtn();
```

Comment:
this can be only one method called login or signIn, and that method can receive two params, the email and pass.